tutorials/2d/canvas_layers.rst

```diff
@@ -66,11 +58,8 @@ Here's a diagram of how it looks:
 CanvasLayers are independent of tree order, and they only depend on
 their layer number, so they can be instantiated when needed.
 
-Performance
------------
+.. note:: CanvasLayers aren't necessary to control the drawing order of
+          nodes. The most optimal way of ensuring that a node is correctly drawn 'in front' or 'behind' others is to                   manipulate the order of the nodes in the scene tree. The topmost nodes are drawn on top of lower ones. 2d nodes             also have a property for controlling their drawing order
+          (see :ref:`Node2D.z_index <class_Node2D_property_z_index>`).
+          
```

Comment:
For consistency, maintain the indentation under the note block. See, for example, `your_first_game.rst` or `2d_lights_and_shadows.rst`

Reply:
I've tried to fix the indentation. Doing my edits within github. Hope it's correct!